### PR TITLE
fix: transaction sign and OP_RETURN + import

### PR DESCRIPTION
**Pull request by Alex-Werner**
## What was done?
- allow .sign() method to auto-sign transaction based on inputs.
- allow importTransaction() to import JSON based data
- allow .sign() to handle cases with OP_RETURN (#160)

## How Has This Been Tested?
Added provided tests

## Breaking Changes
N/A

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dashevo/wallet-lib",
-  "version": "7.15.0",
+  "version": "7.15.1",
```
Comment (shumkov) [inline package.json:3]: We usually separate bump and actual changes into different PRs, otherwise the [conventional-changelog-cli](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-cli) which we use to generate changelogs is not working properly.

**Review by shumkov — CHANGES_REQUESTED** [review]

**Review by antouhou — COMMENTED** [review]
Seems great! I can't wrap my head around why the commits from the #185 ended up in this PR though 🤔

**Comment by antouhou** [comment]
I've updated the 0.16 branch, so now those commits are gone from this PR 👍

File: package.json (hunk 1 above)
Comment (Alex-Werner) [reply package.json]: Seems to be fixed by @antouhou update of 0.16 branch :D

**Review by antouhou — APPROVED** [review]
uTACK 👍
